classes/class-wc-connect-help-provider.php

```diff
@@ -2,20 +2,263 @@
 
 if ( ! class_exists( 'WC_Connect_Help_Provider' ) ) {
 
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_WOOCOMMERCE_VERSION', '2.6' );
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_JETPACK_VERSION', '4.0.2' );
```

Comment:
Technically we require 3.9+ [since that's when Tracks was added](https://github.com/Automattic/jetpack/commit/79b50da6b9af436bafcb42ea4e93d8431e25c1e7), which is the most recent feature of Jetpack we require.